fuels-abigen-macro/tests/test_projects/foo-caller-contract/.gitignore

```diff
@@ -0,0 +1,2 @@
+out
+target
```

Comment:
Could you put this in `fuels-abigen-macro/tests/test_projects/.gitignore` instead?

Reply:
This is what's generated by `forc init` (for every test case I use the whole `forc init` workflow), didn't want to change the default output from `forc init` in case people wanted to play with these test projects 🤔.

Reply:
Got it